packages/sources/svelte/src/components/indicators/VtmnProgressbar/test/VtmnProgressbar.spec.js

```diff
@@ -59,7 +59,6 @@ describe('VtmnProgressbar', () => {
         label: 'unit-test',
```

Comment:
Please add unit tests with the new fonctionnalities :)

Reply:
Done